calibration_guis/src/main_window.cpp

```diff
@@ -0,0 +1,249 @@
+/**
+ * @file /src/main_window.cpp
+ *
+ * @brief Implementation for the qt gui.
+ *
+ * @date February 2011
+ **/
+/*****************************************************************************
+** Includes
+*****************************************************************************/
+
+#include <QtGui>
+#include <QMessageBox>
+#include <iostream>
+#include "../include/calibration_guis/main_window.hpp"
+#include "std_srvs/Trigger.h"
+#include "industrial_extrinsic_cal/cal_srv_solve.h"
+
+/*****************************************************************************
+** Namespaces
+*****************************************************************************/
+
+namespace calibration_guis {
+
+using namespace Qt;
```

Comment:
In general, don't use a full namespace
instead use
using Qt::QWidget;
using Qt::blablabla;
That way, you won't accidentally use Qt::matrixMult() instead of Opencv::matrixMult()